treeherder/model/models.py

```diff
@@ -895,6 +895,7 @@ class FailureLine(models.Model):
     best_classification = models.ForeignKey("ClassifiedFailure",
                                             related_name="best_for_lines",
                                             null=True,
+                                            db_constraint=False,
```

Comment:
(Speaking out loud) 

Looking at the output of `show create table treeherder.failure_line;` this causes a single change, the removal of this line:
```CONSTRAINT `failure_line_best_classification__e328ed7b_fk_classifie` FOREIGN KEY (`best_classification_id`) REFERENCES `classified_failure` (`id`),```

Both before and after, there is a DB level `DEFAULT NULL` for that field which will hopefully mean Django doesn't try and reference it (though we should confirm):
````best_classification_id` bigint(20) DEFAULT NULL,````